uplink/config.go

```diff
@@ -155,9 +155,10 @@ func (c Config) GetPathCipherSuite() storj.CipherSuite {
 
 // GetEncryptionScheme returns the configured encryption scheme for new uploads
 func (c Config) GetEncryptionScheme() storj.EncryptionScheme {
+	const stripesPerBlock = 2
 	return storj.EncryptionScheme{
 		Cipher:    storj.Cipher(c.Enc.DataType),
-		BlockSize: c.GetRedundancyScheme().StripeSize(),
+		BlockSize: c.GetRedundancyScheme().StripeSize() * stripesPerBlock,
```

Comment:
i don't know where else this would matter (tests maybe?) but in your original PR wasn't there more than one place where the block size was set to the stripe size?

Reply:
good point, let me make a run through the tests and check they reflect this change